internal/cltest/mocks.go

```diff
@@ -137,7 +146,12 @@ func (mock *EthMock) Call(result interface{}, method string, args ...interface{}
 			return nil
 		}
 	}
-	return fmt.Errorf("EthMock: Method %v not registered", method)
+
+	err := fmt.Errorf("EthMock: Method %v not registered", method)
+	if mock.strict {
+		log.Fatal(err)
```

Comment:
Optional: put the error inline rather than outside the `if`.

Reply:
So:

```
if mock.strict {
		log.Fatal(fmt.Errorf("EthMock: Method %v not registered", method))
}
return fmt.Errorf("EthMock: Method %v not registered", method)
```

?

Reply:
No, don't do that. Ignore my suggestion.